src/shared/api/session-service/sessionServiceAPI.ts

```diff
@@ -17,6 +17,14 @@ export default class sessionServiceAPI {
         return `${getSessionUrl()}/virtual_study`;
     }
 
+    getPublicVirtualStudyServiceUrl() {
+        //FIXME change url after moving the code to the session controller
```

Comment:
are we fixing this?

Reply:
Yes, here it is https://github.com/cBioPortal/cbioportal-frontend/pull/4923/commits/2aea3c5312d223ff8d105e602a06b19b865e0568